python/ray/data/_internal/stats.py

```diff
@@ -808,6 +813,13 @@ def to_string(self) -> str:
                     fmt(self.collate_time.avg()),
                     fmt(self.collate_time.get()),
                 )
+            if self.finalize_batch_time.get():
+                out += "   * In finalize_fn: {} min, {} max, {} avg, {} total\n".format(
```

Comment:
Users don't know what finalize_fn means. We should rename this to host->device transfer